### PR TITLE
init angular app

### LINKED ISSUE #1: Init angular app
Create a new angular app, using the PRX Styleguide module.

(Probably requires waiting until more Publish components are moved into the Styleguide)

App should:

- [ ] Show navbar, with current logged in user
- [ ] Allow user login
- [ ] Some sort of dashboard, showing podcasts/episodes belonging to user.  And maybe some basic stats for them.

### PR DESCRIPTION
**Pull request by sandikbarr**
Fixes #1

### CODE REVIEW

File: src/app/error/error.service.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import { Injectable, ErrorHandler } from '@angular/core';
```
Comment (sandikbarr) [inline src/app/error/error.service.ts]: TODO: should be a common ErrorService https://github.com/PRX/styleguide.prx.org/issues/52

File: README.md [hunk 2]
```diff
@@ -1,2 +1,70 @@
...
+```
+Metrics will connect to id-staging.prx.org, cms-staging.prx.tech, and your local Castle server.
+
+### Use local `cms`
+To run cms locally, change the `CMS_HOST` in `.env` to `CMS_HOST=cms.prx.dev`.
+
+###  Use local `id`
+To run id locally, change the `AUTH_HOST` in `.env` to `AUTH_HOST=id.prx.dev`.
+
+Next, you will need to create a client application set up, this is easiest to do from the prx.org console:
```
Comment (cavis) [inline README.md]: Should be "the ID console", not the prx.org/v3 console.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,70 @@
...
+``` ruby
+# start a console for prx.org
+cd prx.org
```
Comment (cavis) [inline README.md]: Same here.

File: env-example [hunk 4]
```diff
@@ -0,0 +1,21 @@
+# (1) cp env-example .env
+# (2) add your ENV overrides here
+# (3) profit
+LOCAL_ENV=true
+
+# prx api
+CMS_HOST=cms-staging.prx.tech
```
Comment (cavis) [inline env-example]: `cms.staging` (and yeah, i haven't updating other app's env-examples yet)

File: env-example [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+CMS_TTL=
+
+# prx auth
+AUTH_HOST=id-staging.prx.tech
```
Comment (cavis) [inline env-example]: and `id.staging`

**Comment by cavis** [comment]
+2

**Pull request merged**